sshutil/certificate_test.go

```diff
@@ -26,7 +26,7 @@ func (b *badSigner) Sign(r io.Reader, data []byte) (*ssh.Signature, error) {
 	return nil, fmt.Errorf("an error")
 }
 
-func (b *badSigner) SignWithAlgorithm(r io.Reader, data []byte, algorithm string) (*ssh.Signature, error) {
+func (b *badSigner) SignWithAlgorithm(_ io.Reader, data []byte, algorithm string) (*ssh.Signature, error) {
```

Comment:
linter didn't catch this?

Reply:
https://github.com/smallstep/workflows/pull/31/files -- stopped using this linter for tests midway through generating the PR.